metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughAPITest.java

```diff
@@ -18,6 +18,7 @@
 
 package uk.ac.ebi.ampt2d.metadata.importer.api;
 
+import org.junit.Assert;
```

Comment:
Can be removed.

Reply:
Removed all unused imports, fixed in https://github.com/EBIvariation/metadata-ws/pull/55/commits/0e5f9b50a3fc908dd654a8d64e7aac5919050f3a